src/connection.ts

```diff
@@ -160,6 +169,13 @@ export class Connection extends JSForceConnection {
     return conn;
   }
 
+  private static createReadStreamFromBuffer(buffer: Buffer): fs.ReadStream {
```

Comment:
Let's not do this...

Why not just do something like:

```
import { Readable } from 'stream';
...
Readable.from(buffer);
```

Reply:
I tried transforming the `buffer` into a `ReadableStream` just about every way that I could find without any working. The api seems to only accept types of `fs.ReadStream`. I'm very open to pairing and showing you what I tried and the results they produce